backend/ccm/canvas_api/urls.py

```diff
@@ -5,7 +5,7 @@
 from backend.ccm.canvas_api.section_enrollments_api_handler import CanvasSectionEnrollmentsAPIHandler
 
 urlpatterns = [
-  path('course/<int:course_id>/', CanvasCourseAPIHandler.as_view() , name='course'),
-  path('course/<int:course_id>/sections/', CanvasCourseSectionAPIHandler.as_view() , name='courseSection'),
-  path('sections/students/', CanvasSectionEnrollmentsAPIHandler.as_view() , name='sectionEnrollments'),
```

Comment:
This endpoint is used by 3rd-party Gradebook, if you select a section then this api pull all student enrollments, make sure you have users in the section. You can add me or Matt to the course as enrollment or go to Root --> People and add any users

Reply:
I think this is working, making requests to those URLs was working but I would like some verification.

Reply:
I checked this and was working